recipes/mono-msbuild/all/test_package/conanfile.py

```diff
@@ -0,0 +1,15 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanException
+import os
+
+
+class TestPackageConan(ConanFile):
+    #generators = "cmake"
+
+    def build(self):
+        pass
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            self.output.info("Version:")
```

Comment:
The configuration setting `log.print_run_commands` echos the command of `self.run` commands.
```suggestion
        if not tools.cross_building(self):
```